ComicsWebApp/Data/ComicsDbContext.cs

```diff
@@ -0,0 +1,45 @@
+﻿using ComicsWebApp.Models;
+using Microsoft.EntityFrameworkCore;
+
+namespace ComicsWebApp.Data
+{
+    public class ComicsDbContext: DbContext
+    {
+        public DbSet<Comics> Comics { get; set; }
+        public DbSet<ComicsGenre> ComicsGenres { get; set; }
+        public DbSet<ComicsPages> ComicsPages { get; set; }
+
+        public ComicsDbContext(DbContextOptions<ComicsDbContext> options)
+            : base(options)
+        { }
+
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
```

Comment:
you can set up a length for this short fields on db

Reply:
I meant to not use nvarchar(max ) for filed like Name or Status, but use nvarchar(20)